modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalCardTag.java

```diff
@@ -0,0 +1,821 @@
+		props.put("flushHorizontal", getFlushHorizontal());
+		props.put("flushVertical", getFlushVertical());
+		props.put("href", getHref());
+		props.put("id", getId());
+		props.put("imageAlt", getImageAlt());
+		props.put("imageSrc", getImageSrc());
+		props.put("inputName", getInputName());
+		props.put("inputValue", getInputValue());
+		props.put("labels", getLabels());
+		props.put("labelStylesMap", getLabelStylesMap());
+		props.put("selectable", getSelectable());
+		props.put("selected", getSelected());
+		props.put("stickerCssClass", getStickerCssClass());
+		props.put("stickerIcon", getStickerIcon());
+		props.put("stickerImageAlt", getStickerImageAlt());
+		props.put("stickerImageSrc", getStickerImageSrc());
+		props.put("stickerLabel", getStickerLabel());
+		props.put("stickerShape", getStickerShape());
+		props.put("stickerStyle", getStickerStyle());
+		props.put("symbol", getIcon());
+		props.put("title", getTitle());
+
+		return super.prepareProps(props);
+	}
+
+	@Override
+	protected String processCssClasses(Set<String> cssClasses) {
+		cssClasses.add("card-type-asset");
+
+		if (Validator.isNotNull(_imageSrc)) {
+			cssClasses.add("image-card");
+		}
+		else {
+			cssClasses.add("file-card");
+		}
+
+		if (!_selectable) {
+			cssClasses.add("card");
+		}
+		else {
+			cssClasses.add("form-check");
+			cssClasses.add("form-check-card");
+			cssClasses.add("form-check-top-left");
+		}
+
+		return super.processCssClasses(cssClasses);
+	}
+
+	@Override
+	protected int processStartTag() throws Exception {
+		super.processStartTag();
+
+		JspWriter jspWriter = pageContext.getOut();
+
+		if (_selectable) {
+			jspWriter.write("<div class=\"card\"><div class=\"custom-control ");
+			jspWriter.write("custom-checkbox\"><label");
+
+			if (Validator.isNotNull(_inputName)) {
+				jspWriter.write(" for=\"");
+				jspWriter.write(_inputName);
+				jspWriter.write("\" ");
+			}
+
+			jspWriter.write("><input ");
+
+			if (_selected) {
+				jspWriter.write("checked=\"checked\" ");
+			}
+
+			jspWriter.write("class=\"custom-control-input\" ");
+
+			if (Validator.isNotNull(_inputName)) {
+				jspWriter.write("id=\"");
+				jspWriter.write(_inputName);
+				jspWriter.write("\" ");
+			}
+
+			jspWriter.write("type=\"checkbox\" ");
+
+			if (_disabled) {
+				jspWriter.write("disabled=\"disabled\"");
+			}
+
+			if (Validator.isNotNull(_inputValue)) {
+				jspWriter.write("value=\"");
+				jspWriter.write(_inputValue);
+				jspWriter.write("\" ");
+			}
+
+			jspWriter.write(">");
+			jspWriter.write("<span class=\"custom-control-label\"></span>");
+			jspWriter.write("<div class=\"card-item-first aspect-ratio\">");
+
+			if (Validator.isNotNull(_imageSrc)) {
+				_renderImage(jspWriter);
+			}
+			else {
+				_renderIcon(jspWriter);
+			}
+
+			if (Validator.isNotNull(_stickerLabel)) {
+				StickerTag sticker = new StickerTag();
+
+				sticker.setDisplayType(_stickerStyle);
+				sticker.setLabel(_stickerLabel);
```

Comment:
Let me call you about this via slack because I have a doubt

Reply:
[This commit](https://github.com/liferay-frontend/liferay-portal/pull/448/commits/1c1be56102a1ff7c92e337a2810d78bc077f1f37) is the one I want us to talk about

Reply:
@carloslancha since we [talked](https://liferay.slack.com/archives/G3JBR21HA/p1604505906307400) about undeprecating `StickerTag`'s `label` attribute and that it is OK I pushed [94e28d7](https://github.com/liferay-frontend/liferay-portal/pull/448/commits/94e28d7117334dd62f0f746994561ec23f5ee00f)

I also checked that the sticker tag can be used like this:

```jsp
<clay:sticker
    label="PDF"
    position="top-left"
    style="danger"
/>
```

as well as like this:

```jsp
<clay:sticker>
   <img class="sticker-circle sticker-img" src="/image/layout_set_logo?img_id=39974&t=1604581407252" />
</clay:sticker>
```